Tests/Common/Securities/Futures/FuturesExpiryFunctionsTests.cs

```diff
@@ -16,12 +16,189 @@
 using System;
 using NUnit.Framework;
 using QuantConnect.Securities.Future;
+using System.Collections.Generic;
+using System.Linq;
+using System.Xml;
+using System.Xml.Serialization;
+using System.IO;
 
 namespace QuantConnect.Tests.Common.Securities.Futures
 {
     [TestFixture]
     public class FuturesExpiryFunctionsTests
     {
+        /// <summary>
+        /// Dates for Termination Conditions of futures
+        /// </summary>
+        public class Dates
```

Comment:
Okay, I will also follow that convention. Unfortunately yes it needs to
be public to use XmlSerializer. Otherwise, it gives the following error:

SetUp : System.InvalidOperationException :
QuantConnect.Tests.Common.Securities.Futures.FuturesExpiryFunctionsTests+Item
is inaccessible due to its protection level. Only public types can be
processed.